packages/x-adapter/src/mappers/__tests__/combine-mappers.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { DeepPartial } from '@empathyco/x-utils';
+import { Mapper } from '../../types/mapper.types';
+import { combineMappers } from '../combine-mappers';
+
+describe('combineMappers tests', () => {
+  const testMapper: Mapper<TestFrom, DeepPartial<TestTo>> = ({ query, count }) => {
+    return {
+      q: query,
+      c: count
+    };
+  };
+  const extraMapper: Mapper<TestFrom, DeepPartial<TestTo>> = ({ extra }) => {
+    return {
+      e: extra
+    };
+  };
+
+  it('combines a single mapper', () => {
+    const combinedMappers = combineMappers<TestFrom, TestTo>(testMapper);
+    expect(combinedMappers({ query: 'patata', count: 0, extra: 'croqueta' }, {})).toEqual({
+      q: 'patata',
+      c: 0
+    });
+  });
+
+  it('combines multiple mappers', () => {
+    const combinedMappers = combineMappers<TestFrom, TestTo>(testMapper, extraMapper);
+    expect(combinedMappers({ query: 'patata', count: 0, extra: 'croqueta' }, {})).toEqual({
+      q: 'patata',
+      c: 0,
+      e: 'croqueta'
+    });
+  });
```

Comment:
Should we test the context parameter too?

Reply:
Good point, I've updated the tests.